src/hdmf/build/map.py

```diff
@@ -1387,6 +1410,8 @@ def get_container_cls(self, **kwargs):
                 if not spec.is_inherited_spec(field_spec):
                     fields[k] = field_spec
             d = self.__get_cls_dict(parent_cls, fields)
+            if '__clsconf__' in d:
```

Comment:
Do not assume `__clsconf__` indicates _MultiContainerInterface_

Reply:
```suggestion
            if any(hasattr(field_spec, 'quantity') and field_spec.quantity in ('*', '+') for field in addl_fields):
```